essentials/custom_node_more_on_inputs.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: "Hidden and Flexible inputs"
+---
+
+## Hidden inputs
+
+Alongside the `required` and `optional` inputs, which create corresponding inputs or widgets on the client-side,
+there are three `hidden` input options which allow the custom node to request certain information from the server.
+
+These are accessed by returning a value for `hidden` in the `INPUT_TYPES` `dict`, with the signature `dict[str,str]`,
+containing one or more of `PROMPT`, `EXTRA_PNGINFO`, or `UNIQUE_ID`
+
+```python
+@classmethod
+def INPUT_TYPES(s):
+    return {
+        "required": {...},
+        "optional": {...},
+        "hidden": {
+            "prompt": "PROMPT", 
```

Comment:
How would you feel about arranging these and the headings below in the same order?
Either alphabetical or by perceived importance?

Reply:
Yup. Pushing update in a minute with this and other tweaks.